scripts/irods/test/test_quotas.py

```diff
@@ -97,3 +97,54 @@ def test_filter_out_groups_when_selecting_user__issue_3507(self):
 
         finally:
             self.admin.assert_icommand(['iadmin', 'rmgroup', 'test_group_3507'])
+
+    def test_physical_quota_case_10_11_12(self):
+        pep_map = {
+            'irods_rule_engine_plugin-irods_rule_language': textwrap.dedent('''
+                acRescQuotaPolicy {
+                    msiSetRescQuotaPolicy("on");
+                }
+            '''),
+            'irods_rule_engine_plugin-python': textwrap.dedent('''
+                def acRescQuotaPolicy(rule_args, callback, rei):
+                    callback.msiSetRescQuotaPolicy('on')
+            ''')
+        }
+
+        try:
+            with temporary_core_file() as core:
+                core.add_rule(pep_map[self.plugin_name])
+                IrodsController().reload_configuration()
+
+                group_name = 'test_group_case_10_11_12'
+                file_name = 'test_file_case_10_11_12'
+                resc_name = 'ufs0_quota_case_10_11_12'
+                lib.make_file(file_name, 4096, contents='arbitrary')
+                lib.create_ufs_resource(self.admin, resc_name)
+
+                self.admin.assert_icommand(['iadmin', 'mkgroup', f'{group_name}_1'])
+                self.admin.assert_icommand(['iadmin', 'mkgroup', f'{group_name}_2'])
+                self.admin.assert_icommand(['iadmin', 'sgq', f'{group_name}_1', resc_name, '5000'])
+                self.admin.assert_icommand(['iadmin', 'sgq', f'{group_name}_2', resc_name, '10000'])
+
+                self.admin.assert_icommand(['iadmin', 'atg', f'{group_name}_1', self.admin.username])
+                self.admin.assert_icommand(['iadmin', 'atg', f'{group_name}_2', self.admin.username])
+
+                self.admin.assert_icommand(['iput', '-R', self.testresc, file_name])
+
```

Comment:
not strictly required - but could be cleaner to also have `iadmin cu` here -  to make sure the totals are up to date.